test/jdk/javax/swing/JPasswordField/bug4382819.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2000, 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Color;
+import java.awt.FlowLayout;
+import java.awt.Point;
+import java.awt.Robot;
+import javax.swing.JFrame;
+import javax.swing.JPasswordField;
+import javax.swing.SwingUtilities;
+
+/*
+ * @test
+ * @bug 4382819
+ * @key headful
+ * @summary Tests the correctness of color used for the disabled passwordField.
+ * @run main bug4382819
+ */
+
+public class bug4382819 {
+    static JFrame mainFrame;
+    static volatile Point p;
+    static JPasswordField passwordField;
+
+    public static void main(String[] args) throws Exception {
+        try {
+            Robot robot = new Robot();
+            robot.setAutoDelay(250);
+            SwingUtilities.invokeAndWait(() -> createTestUI());
+            robot.waitForIdle();
```

Comment:
Add some delay after testUI ?

```suggestion
            robot.waitForIdle();
            robot.delay(1000);
```